src/frontend/packages/core/src/shared/components/cards/card-cf-space-details/card-cf-space-details.component.html

```diff
@@ -12,11 +12,15 @@
           <app-metadata-item label="Quota Definition Name">
             <ng-container
               *ngIf="(cfSpaceService.spaceQuotaDefinition$ | async) as spaceQuotaDefinition; else fallBackQuota">
-              {{ spaceQuotaDefinition.name }}
+              <a routerLink="../space-quota">
+                {{ spaceQuotaDefinition.name }}
+              </a>
             </ng-container>
             <ng-template #fallBackQuota>
-              None assigned<span *ngIf="cfSpaceService.quotaDefinition$ | async as quotaDefinition"> (showing
-                organization's '{{ quotaDefinition.name }}')</span>
+              None assigned
+              <span *ngIf="cfSpaceService.quotaDefinition$ | async as quotaDefinition">(showing
+                organization's <a routerLink="../quota">{{ quotaDefinition.name }}</a>)
```

Comment:
This should be a link to the org quota page rather than displaying the org quota in the space quota page. Soimthing like "(View the organization <link>quota</link>)"